src/tokens/line-heights.json

```diff
@@ -0,0 +1,16 @@
+{
+  "line-height": {
+    "base": {
+      "value": "1.5"
+    },
+    "title": {
```

Comment:
Since these `title` and `tight` are the same value, can we ditch `title` all together and replace all instances where we're using `title` with `tight`?

I think it will be more straightforward to just have the three values for now instead of duplicating `tight` and `title` since `title` doesn't necessarily match the naming convention.

Reply:
Agreed! Thanks :)

Reply:
@levimcg Alright, I've removed the `.rvt-lh-title` class. ✂️

Reply:
Awesome. I think we can go ahead an remove it from the `line-heights.json` file as well and we'll be good to go!